src/main/java/net/mcreator/ui/modgui/EnchantmentGUI.java

```diff
@@ -91,51 +92,51 @@ public EnchantmentGUI(MCreator mcreator, ModElement modElement, boolean editingM
 		JPanel selp = new JPanel(new GridLayout(11, 2, 100, 2));
 		selp.setOpaque(false);
 
-		selp.add(HelpUtils.wrapWithHelpButton(this.withEntry("enchantment/name"), new JLabel("Enchantment name: ")));
+		selp.add(HelpUtils.wrapWithHelpButton(this.withEntry("enchantment/name"), new JLabel(L10N.t("elementgui.enchantment.name"))));
 		selp.add(name);
 
-		selp.add(HelpUtils.wrapWithHelpButton(this.withEntry("enchantment/type"), new JLabel("Enchantment type: ")));
+		selp.add(HelpUtils.wrapWithHelpButton(this.withEntry("enchantment/type"), new JLabel(L10N.t("elementgui.enchantment.type"))));
 		selp.add(type);
 
 		selp.add(
-				HelpUtils.wrapWithHelpButton(this.withEntry("enchantment/rarity"), new JLabel("Enchantment rarity: ")));
+				HelpUtils.wrapWithHelpButton(this.withEntry("enchantment/rarity"), new JLabel(L10N.t("elementgui.enchantment.rarity"))));
 		selp.add(rarity);
 
 		selp.add(HelpUtils
-				.wrapWithHelpButton(this.withEntry("enchantment/min_level"), new JLabel("Minimal supported level: ")));
+				.wrapWithHelpButton(this.withEntry("enchantment/min_level"), new JLabel("elementguielementgui.enchantment.min_level")));

```

Comment:
Please remove the extra "elementgui" at the beginning of this one: "elementguielementgui.enchantment.min_level"

Reply:
Actually, this is not even translation but a label with the text "elementguielementgui.enchantment.min_level"